### PR TITLE
Add 'active' prop to Button to avoid having to use 'pt-active' in className

### LINKED ISSUE #710: active prop on Button
Hello,

What's the general thought around an `active` prop on a `Button`? Currently I'm using `classnames` to set `<Button className={cx({'pt-active': var})}>` and it would be nice to just be able to use `<Button active={var}>`

Totally understand if it doesn't make sense to set the button's active state via a prop.

Thanks! 
Drew

### PR DESCRIPTION
**Pull request by DrewDennison**
Fixes #710

This is my first PR so please let me know if I need to update docs or do anything else.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @DrewDennison! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/core/src/components/button/abstractButton.tsx [hunk 1]
```diff
@@ -29,6 +29,12 @@ export interface IButtonProps extends IActionProps {
     loading?: boolean;
 
     /**
+     * If set to `true`, the button will display in an active state. This is equivalent to setting `pt-active` via className.
```
Comment (giladgray) [inline packages/core/src/components/button/abstractButton.tsx]: this line is too long, fails linting. please wrap after the period.

also, how about "Whether the button will..." ?

Comment (DrewDennison) [inline packages/core/src/components/button/abstractButton.tsx]: Fixed the line length issue. 

Would you rather 'If set to `true`, the button will display in an active state.' or 'Whether the button will display in an active state.' or something else?

Comment (DrewDennison) [inline packages/core/src/components/button/abstractButton.tsx]: Maybe 'Controls whether the button will display in an active state'?

Comment (cmslewis) [inline packages/core/src/components/button/abstractButton.tsx]: @DrewDennison the Blueprint pattern is something like your second offering above:

image

 "Whether the button is active." is probably the most in line with the above pattern, but then it's not necessarily clear what "active" means at a glance. @giladgray may have thoughts.

**Comment by giladgray** [comment]
🎉  thank you @DrewDennison!

**Pull request merged**